### PR TITLE
octanify instructorgroup-details component.

### PR DESCRIPTION
**Pull request by stopfstedt**

### CODE REVIEW

File: app/components/instructorgroup-details.js [hunk 1]
```diff
@@ -1,24 +1,22 @@
-import Component from '@ember/component';
+import Component from '@glimmer/component';
+import { dropTask } from 'ember-concurrency';
 
-export default Component.extend({
-  tagName: '',
+export default class InstructorgroupDetailsComponent extends Component {
+  @dropTask
+  *addUser(user) {
+    const users = yield this.args.instructorGroup.users;
```
Comment (jrjohnson) [inline app/components/instructorgroup-details.js:7]: I'm fairly certain this was a workaround for old Ember Data and only one side of this relationship needs to be changed and saved.

**Review by jrjohnson — APPROVED** [review]
One comment, that can be fixed later.

File: app/components/instructorgroup-details.js (hunk 1 above)
Comment (stopfstedt) [reply app/components/instructorgroup-details.js:7]: handled. :heavy_check_mark:

**Review by jrjohnson — APPROVED** [review]

**Comment by dartajax** [comment]
I clicked tested a bit. It seems fine - gonna merge.

**Pull request merged**